assets/src/dashboard/app/views/myStories/content/listView/index.js

```diff
@@ -347,11 +338,17 @@ export default function StoryListView({
                 data-testid={`story-list-item-${story.id}`}
               >
                 <TablePreviewCell>
-                  <PreviewContainer>
-                    <PreviewErrorBoundary>
-                      <PreviewPage page={story.pages[0]} pageSize={pageSize} />
-                    </PreviewErrorBoundary>
-                  </PreviewContainer>
+                  <PreviewImage
+                    as={!story.featuredMediaUrl ? 'div' : 'img'}
```

Comment:
@BrittanyIRL same thing here with the `div`.

Reply:
yup, i realized i forgot that, i'll circle back and update this later today. appreciate the eyes